models/grading/schema.yml

```diff
@@ -61,3 +74,63 @@ models:
       - name: grade_bucket
         description: "A displayable value of grades sorted into 10% buckets. Useful for grouping grades together to show high-level learner performance"
         data_type: String
+
+  - name: fact_learner_course_grade
+    description: "One record per learner per course for the most recent grade"
+    columns:
+      - name: org
+        data_type: string
+        description: "The organization that the course belongs to"
+      - name: course_key
+        data_type: string
+        description: "The course key for the course"
+      - name: actor_id
+        data_type: string
+        description: "The xAPI actor identifier"
+      - name: course_grade
+        data_type: float64
+        description: "The most recent grade for the learner"
+
+  - name: fact_learner_course_status
+    description: "One record per learner per course for the most recent approving status"
```

Comment:
I'm not sure what "approving status" is here? Is this pass/fail?

Reply:
Yes, it's passed or failed

Reply:
Can you clarify that in the description?

Reply:
done